Kitodo-DataManagement/src/main/java/org/kitodo/data/elasticsearch/index/Indexer.java

```diff
@@ -85,40 +82,35 @@ public String performSingleRequest(T baseIndexedBean, S baseType) throws IOExcep
      * @param beanId
      *            response from the server
      */
-    public String performSingleRequest(Integer beanId) throws IOException, CustomResponseException {
+    public void performSingleRequest(Integer beanId) throws IOException, CustomResponseException {
         IndexRestClient restClient = initiateRestClient();
-        String response;
 
         if (method == HTTPMethods.DELETE) {
-            response = String.valueOf(restClient.deleteDocument(beanId));
+            restClient.deleteDocument(beanId);
```

Comment:
Why are you ignoring return value of `deleteDocument` method? Previous behaviour was even wrong as this method is returning a boolean value, but this is an other story.

Reply:
Boolean is not returned anymore.